docs/pages/documentation/10-resources/faq.md

```diff
@@ -28,31 +28,14 @@ There are lots of ways you can get involved! Please take a look at our [contribu
 
 ## Bugs and strange behaviour
 
-### Why does Grakn hang when I try to start it?   
-
-I am running `grakn.sh start` but it hangs on `Starting Cassandra`. Why?
-
-This may be because you have cloned the Grakn repo into a directory which has a space in its name (e.g. `/grakn test`). You can build our code successfully, but when you start `grakn.sh`, it hangs because Cassandra needs you to have single word pathnames. Remove the spaces (e.g. `/grakn_test`) and try again.
```

Comment:
If this is still a problem, we should keep it around (without necessarily mentioning Cassandra). It can really trip you up.

Reply:
We can use the same nomenclature that the config files have, eg. storage/queue.